.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    if: ${{ github.repository_owner == 'cloudflare' }}
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js 16.7
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.7
+      
+      - uses: actions/cache@v2
+        id: node-modules-cache
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+      
+      - name: Install NPM   
+        run: npm install
+
+      - name: Check Docs
+        run: npm run export:docs
+      - name: Check Overrides
+        run: npm run export:overrides
+
+      - name: Run Type Checking 
```

Comment:
Should we call prettier check here as well

Reply:
Yes, there could be cases that it catches changes that PR workflow was bypassed.